Source/Core/VideoBackends/D3D/NativeVertexFormat.cpp

```diff
@@ -124,6 +127,11 @@ ID3D11InputLayout* D3DVertexFormat::GetInputLayout(D3DBlob* vs_bytecode)
   if (m_layout)
```

Comment:
Can we make the assumption that pointer sized writes are atomic? Otherwise you might be reading a half-written value there. Might be worth just making this explicitly atomic.